adrs/0001-pseudo-private-custom-properties.md

```diff
@@ -37,23 +37,23 @@ Defining a structure for component styles using pseudo-private custom properties
     @cn-transition-duration: 100ms;
     // BASE COMPONENT-SPECIFIC CUSTOM PROPERTIES
     // --_{component-abbreviation}-{property-shorthand}: {value};
-    --_cn-bg: var(--blue-500);
```

Comment:
I figured I'd change _every_ color reference, including ones here, since it would make a find and replace easy in my follow-up branch. I don't expect anyone to reference this ADR in this state, so figured it's fine for the sake of consistency.